3.11/release-notes-new-features311.md

```diff
@@ -55,3 +55,21 @@ Example: `arangod --rocksdb.total-write-buffer-size 2TiB`
 
 See [Suffixes for numeric options](administration-configuration.html#suffixes-for-numeric-options)
 for details.
+
+
+Cursor API
+----------
+
+### Retry request for last batch
+
+The cursor API can now receive a retry request to retrieve the response for the
```

Comment:
Same text as in https://github.com/arangodb/docs/pull/1234/files#diff-61ead29dabbd5fe1f434b98fa9483996986bc5570ce39d9dfb2cf5642ea0e65aR66.
The same comments apply here.